esmvalcore/cmor/_fixes/CMIP6/BCC_CSM2_MR.py

```diff
@@ -0,0 +1,66 @@
+# pylint: disable=invalid-name, no-self-use, too-few-public-methods
+"""Fixes for BCC-CSM2-MR."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+
+
+class allvars(Fix):
+    """Common fixes to all vars"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes error in time coordinate, sometimes contains trailing zeros
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.CubeList
+
+        """
+        for cube in cubes:
+            try:
+                old_time = cube.coord('time')
+            except iris.exceptions.CoordinateNotFoundError:
+                return cubes
```

Comment:
```suggestion
                continue
```

Reply:
It would be better to just continue, another cube in the list might contain a time coordinate

Reply:
Changed as suggested.